### PR TITLE
Fix legacy integration tests

### PR DESCRIPTION
**Pull request by chrisburr**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -12,6 +12,6 @@ coverage:
 
 codecov:
   notify:
-    after_n_builds: 2
+    after_n_builds: 7
```
Comment (chaen) [inline codecov.yml:15]: why 7 ?

File: containers/client/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,8 @@
 FROM ghcr.io/diracgrid/diracx/client-base
 
-# Needs to specify how we install diracx
-# * from a local wheel package: PACKAGES_TO_INSTALL=/path/to/diracx.whl
-# * from pypi "diracx==v0.a.b"
-ARG PACKAGES_TO_INSTALL
+ARG EXTRA_PACKAGES_TO_INSTALL
```
Comment (chaen) [inline containers/client/Dockerfile:3]: we should have a default here

File: codecov.yml (hunk 1 above)
Comment (chrisburr) [reply codecov.yml:15]: 6 unit tests + the one with the demo

File: containers/client/Dockerfile (hunk 2 above)
Comment (chrisburr) [reply containers/client/Dockerfile:3]: Just `DIRAC` then?

Comment (chaen) [reply containers/client/Dockerfile:3]: and all the `diracx` modules (not only for this DOckerifle)

**Review by chaen — APPROVED** [review]

**Pull request merged**